Assets/LeapMotion/Core/Scripts/LeapTestProvider.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Leap.Unity {
+
+  public class LeapTestProvider : LeapProvider {
+
+    public Frame frame;
+    public override Frame CurrentFrame { get { return frame; } }
+    public override Frame CurrentFixedFrame { get { return frame; } }
+
+    [Header("Runtime Basis Transforms")]
+
+    [Tooltip("At runtime, if this Transform is non-null, the LeapTestProvider will "
+           + "create a test-pose left hand at this transform's position and rotation."
+           + "Setting this binding to null at runtime will cause the hand to disappear "
+           + "from Frame data, as if it stopped tracking.")]
+    public Transform leftHandBasis;
+    private Hand _leftHand = null;
+    private Hand _cachedLeftHand = null;
+
+    [Tooltip("At runtime, if this Transform is non-null, the LeapTestProvider will "
+           + "create a test-pose right hand at this transform's position and rotation."
+           + "Setting this binding to null at runtime will cause the hand to disappear "
+           + "from Frame data, as if it stopped tracking.")]
+    public Transform rightHandBasis;
+    private Hand _rightHand = null;
+    private Hand _cachedRightHand = null;
+
+    void Awake() {
+      _cachedLeftHand = TestHandFactory.MakeTestHand(isLeft: true);
+      _cachedLeftHand.TransformToUnityUnits(); // Note: sucks that this has to be here
```

Comment:
How about we add it as an enum or a bool switch in the Make method instead then?

Reply:
I think that hides the problem one level deeper without addressing it fully, but the "not this PR" syndrome is real..

I'll add a switch in the method call for now but will want to make more changes in a later PR.